package chap10;

public class ExceptionEx3 {
	public static void main(String[] args) {
		try {
			System.out.println(args[0]);
		}catch(ArithmeticException e){
			System.out.println("0으로 나누지 마세요");
		}catch(e e){
			
		}
		System.out.println("프로그램 종료");
	}

}
